diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index 941cb6b9190ec56aee9b8b2af7791b571da6cb4e..bba2cc0ebabcd910cdcb226b0df93a10aa3b5dd7 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -7855,6 +7855,7 @@ const FeatureEntry kFeatureEntries[] = {
      FEATURE_VALUE_TYPE(features::kDragAndDrop)},
 #endif  // BUILDFLAG(IS_ANDROID)
 
+     BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
     {"autofill-enable-update-virtual-card-enrollment",
      flag_descriptions::kAutofillEnableUpdateVirtualCardEnrollmentName,
      flag_descriptions::kAutofillEnableUpdateVirtualCardEnrollmentDescription,
